app/models/form526_submission.rb

```diff
@@ -338,6 +337,8 @@ def personalization_parameters(first_name)
     }
   end
 
+  private
```

Comment:
I made `#personalization_parameters` public so that the bugfix actually works. It's a small and uneventfully read-only method. We will refactor this in a later PR.